### PR TITLE
feat: add get departments/offices endpoints

### PR DESCRIPTION
**Pull request by BaBa0525**
# Changes

- add get departments by category id
- add get offices by department id

### CODE REVIEW

**Comment by BaBa0525** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #14** <a href="https://app.graphite.dev/github/pr/GymSquad/archive-api/14?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #15** <a href="https://app.graphite.dev/github/pr/GymSquad/archive-api/15?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #16** <a href="https://app.graphite.dev/github/pr/GymSquad/archive-api/16?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/GymSquad/archive-api/16?utm_source=stack-comment).

File: app/categories/schemas.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseModel, ConfigDict
+
+
+class AllCategoriesIdResponse(BaseModel):
+    ids: list[str]
```
Comment (BaBa0525) [inline app/categories/schemas.py:5]: ids 會很怪嗎 ouo

Comment (alan910127) [reply app/categories/schemas.py:5]: 應該還好，不過你也可以像拿 office 一樣回多一點東西

**Review by alan910127 — COMMENTED** [review]
有 merge conflicts oao 你是不是要重新 sync 一下

**Comment by BaBa0525** [comment]
> 有 merge conflicts oao 你是不是要重新 sync 一下

image

**Review by alan910127 — APPROVED** [review]

**Pull request merged**